ja/contributing/backwards-compatibility.rst

```diff
@@ -54,7 +54,7 @@ CakePHP が提供するクラスを構成する、アプリケーションから
     CakePHP のいくつかのクラスは ``@internal`` API ドキュメントタグがつけられています。
     これらのクラスは堅牢 **ではなく** 、後方互換性は約束されていません。
 
-マイナーリリースでは、クラスに新しいメソッドが追加されることがあり、
+マイナーリリース (3.x.0) では、クラスに新しいメソッドが追加されることがあり、
 既存のメソッドは新しい引数が追加されることもあります。
 新しい引数は必ずデフォルト値を持ちますが、異なる引数の形式でメソッドを
 オーバーライドしていると致命的な (**fatal**) エラーになることがあります。
```

Comment:
'fatal' seems to be removed from the original text. Actually, it would be E_STRICT level errors, except for interfaces.